packages/api/src/controllers/stream.ts

```diff
@@ -1057,9 +1064,21 @@ app.put(
       });
     }
 
+    let profiles = req.config.defaultStreamProfiles;
```

Comment:
Can you name this `defaultProfiles` (and change L1081 to `profiles: defaultProfiles`) so this is clearer? Otherwise the request will still override these and that can be misleading.